src/Sulu/Bundle/SecurityBundle/Resources/public/js/tests/unit/security-checker.js

```diff
@@ -0,0 +1,55 @@
+define(function(require) {
+    var registerSuite = require('intern!object');
+    var assert = require('intern/chai!assert');
+    var securityChecker = require('../../services/security-checker');
+
+    registerSuite({
+        name: 'securityChecker',
+
+        'grants permission to objects without permission data': function() {
```

Comment:
That's something we should discuss. What should the keys for this lines look like? I like this notation, because it clearly says what the test should be doing, without duplicating information across method name and comment. /cc @chirimoya @wachterjohannes